src/main/java/org/wso2/siddhi/plugins/idea/completion/SiddhiKeywordsCompletionContributor.java

```diff
@@ -0,0 +1,245 @@
+                            .getParentOfType(prevVisibleSibling, DefinitionElementNode.class) != null) {
+                        addValueTypesAsLookups(result);
+                        return;
+                    }
+                    //suggestions after a trigger name node AT keyword
+                    if (PsiTreeUtil.getParentOfType(prevVisibleSibling, TriggerNameNode.class) != null) {
+                        addAtKeyword(result);
+                        return;
+                    }
+                    if (prevPreVisibleSibling != null) {
+                        //Handling suggestions in a query
+                        if (PsiTreeUtil.getParentOfType(element, ExecutionElementNode.class) != null) {
+                            executionElementRelatedKeywordCompletion(result, element, prevVisibleSibling,
+                                    prevVisibleSiblingElementType, prevPreVisibleSibling);
+                            return;
+                        }
+                        //after AT in a Trigger definition, suggest EVERY keyword
+                        if (prevVisibleSiblingElementType == SiddhiTypes.AT && PsiTreeUtil.getParentOfType
+                                (prevPreVisibleSibling, TriggerNameNode.class) != null) {
+                            addEveryKeyword(result);
+                            return;
+                        }
+                        //Window definitions suggestions
+                        if (PsiTreeUtil.getParentOfType(element, WindowDefinitionNode.class) != null) {
+                            windowDefinitionRelatedKeywordCompletion(result, element, prevVisibleSibling,
+                                    prevVisibleSiblingElementType);
+                        }
+                        //suggestions related to function definition
+                        functionDefinitionRelatedKeywordCompletion(result, element, prevPreVisibleSibling,
+                                prevVisibleSiblingElementType);
+                        //TODO: add aggregation definition
+                    }
+                    //Adding suggestions after a comment
+                    if (prevVisibleSibling instanceof PsiComment && (prevVisibleSibling.getParent() instanceof
+                            SiddhiFile
+                            || prevVisibleSibling.getParent() instanceof ParseNode
+                            || prevVisibleSibling.getParent().getParent() instanceof ParseNode
+                            || prevVisibleSibling.getParent() instanceof SiddhiAppNode)) {
+                        addInitialTypesAsLookups(result);
+                    }
+                }
+            }
+        }
+    }
+
+    //TODO: Restructure definition related completions
+    private void windowDefinitionRelatedKeywordCompletion(@Nonnull CompletionResultSet result, PsiElement element,
+                                                          PsiElement prevVisibleSibling, IElementType
```

Comment:
Move the `IElementType` parameter declaration to a new line. Fix in other similar places as well.

Reply:
Fixed in commit 3aa87a6